### ISSUE TITLE
Allow fine-tuning number of characters changed

### ISSUE THREAD

**Issue by Gitoffthelawn** [sneakypete81/updatescanner#366]
Currently, the Update Scanner UI implements large steps when specifying the number of characters that must be different for a change to be identified.

Can you allow the slider to be completely adjustable, so the user can specify an exact (or close to exact) number of characters that must be different?  If not, can you add an input field, where the user can enter a discrete value?

Thanks!

**Comment by sneakypete81** [comment]
Could you please provide a few more details why this is needed?

**Comment by Gitoffthelawn** [comment]
Yes. I use Update Scanner to monitor websites in order to detect when new releases of software are available.  Sometimes small changes need to be ignored, but their size is between the broad ranges currently provided by the Update Scanner slider.

By allowing the slider to be completely adjustable, or by providing an input field, Update Scanner will be able to detect changes more accurately, while ignoring false positives.

**Comment by sneakypete81** [comment]
Thanks, that's helpful. As you suggest, perhaps an easier alternative would be to add more steps to the slider.

**Comment by Gitoffthelawn** [comment]
You're welcome. Thanks for being open to changes.  Can the slider easily be made to be infinitely adjustable?  You see that a lot in color selector sliders (CYMK/RGB), so I wonder if it's possible here.

**Comment by Mannshoch** [comment]
May add change the currently set count field editable so you could either slide or enter a manual value.
